docs/api/hooks/useFetcher.md

```diff
@@ -42,13 +53,12 @@ function SomeComponent() {
 ## Signature
 
 ```tsx
-useFetcher(options): FetcherWithComponents
+useFetcher<T = any>({ key, }: {})
```

Comment:
Something seems off in the `getSignature` code when there are comments in the types 🤔

Reply:
That's a bummer - but we shouldn't need to inline document them anymore since we're now doing it at the `@params` tag

Reply:
I'll make these updates too 👍